packages/protocol/contracts/stability/StableToken.sol

```diff
@@ -489,6 +490,7 @@ contract StableToken is
     uint256 units = _valueToUnits(inflationState.factor, value);
     totalSupply_ = totalSupply_.sub(units);
     balances[from] = balances[from].sub(units);
+    emit Transfer(from, address(0), value);
```

Comment:
Can we delete `debitFrom`/`creditTo`?

Reply:
Yeah, going to cleanup the other PR too.